$OpenBSD: patch-CMakeLists_txt,v 1.1 2014/07/09 20:03:24 zhuk Exp $
Configure file instead of blind copying, see the
patch-QMobipocketConfig_cmake for details.
--- CMakeLists.txt.orig	Tue Nov  5 00:42:18 2013
+++ CMakeLists.txt	Tue Jul  8 12:20:54 2014
@@ -18,7 +18,9 @@ if(STRIGI_FOUND)
   macro_optional_add_subdirectory(strigi)
 endif()
 
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/QMobipocketConfig.cmake
+configure_file(QMobipocketConfig.cmake QMobipocketConfig.cmake @ONLY)
+
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/QMobipocketConfig.cmake
         DESTINATION ${LIB_INSTALL_DIR}/cmake/QMobipocket )
 
 macro_display_feature_log()
